data/avro-schema/src/main/resources/avro/net/corda/data/interop/InteropIdentity.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "record",
+  "name": "InteropIdentity",
+  "namespace": "net.corda.data.interop",
+  "fields": [
+    {
+      "doc": "Interop group ID.",
+      "name": "groupId",
+      "type": {
+        "type": "string",
+        "logicalType": "uuid"
+      }
+    },
+    {
+      "doc": "Interop identity x500 name.",
+      "name": "x500Name",
+      "type": "string"
+    },
+    {
+      "doc": "Holding Identity of hosting Virtual Node.",
+      "name": "hostingVnode",
+      "type": "string"
+    }
```

Comment:
@jmacmahonr3 This is the only part I think needs another look.

Reply:
Ah yeah ... 
I don't think it's a blocking issue ... 
We can update the names as we go

Reply:
The doc and name are contadicting: "Holding Identity of... " vs "hostingVnode", so what is actually in this field? Is there a Holding identity or Vnode short hash?